### ISSUE TITLE
bulky dependency on Tkinter

### ISSUE THREAD

**Issue by clusterfudge** [buriburisuri/speech-to-text-wavenet#8]
Probably coming in through matplotlib, but it would be nice for this not to be a requirement. Speech recognition should be a relatively headless exercise, and Tkinter does not cleanly install into virtualenvs at this time.

**Comment by buriburisuri** [comment]
This project does not require matplotlib or TKinter.  I don't know why this occur.  Would you tell me your environment? ( Windows or Ubuntu  etc ? )

**Comment by buriburisuri** [comment]
Oh, I find that librosa requires matplotlib.  Sorry but librosa is mandatory for this project.

**Comment by clusterfudge** [comment]
can you explain how librosa is used? Perhaps there's an alternative, or we could carve out the portion needed into a librosa-lite library.

**Comment by buriburisuri** [comment]
Thank you for your information. I'll try librosa-lite

**Comment by clusterfudge** [comment]
To be clear, I don't think librosa-lite actually exists; I was suggesting it as a thing that could be created to suit our needs (it seems like the functionality that doesn't require UI components should be able to be separated from that which does).

**Comment by buriburisuri** [comment]
Oh... it doesn't exist.  Thank you for your information.
If know whether there is any option not to load matplotlib in the 'librosa' library, please let me know.

**Comment by rahulsom** [comment]
https://github.com/librosa/librosa/issues/343 seems relevant. It appears that librosa has been modularized and display has been separated. However that's part of 0.5 which is not yet released.

**Comment by buriburisuri** [comment]
@rahulsom  Thank you for your information.  I'll update librosa dependency when 0.5 is released.

**Comment by akademi4eg** [comment]
@rahulsom @buriburisuri Yeah, the problem is in version of librosa. There is also a workaround for version 0.4
https://groups.google.com/forum/#!topic/librosa/Gk2O9QdkBAc
